src/components/Post/Card.tsx

```diff
@@ -19,10 +21,13 @@ export const postCardTestIds = {
   date: 'post-card__date',
 };
 
-const PostCard = ({ post }: PostCardProps) => {
+const PostCard = ({ post, itemsRowWise = false }: PostCardProps) => {
   return (
     <div
-      className="card bg-base-200 shadow-xl my-8 rounded-lg"
+      className={classNames(
+        { 'w-[48%]': itemsRowWise },
+        'card bg-base-200 shadow-xl my-8 rounded-lg'
+      )}
       data-test-id={postCardTestIds.root}
     >
       <div className="px-8 pt-8 pb-4">
```

Comment:
@tyrro Can you help me fix this one to just `p-8`?
As we don't have any design, I think we can merge those classes into 1 😊

Reply:
sure, done in f2c6dc55159a9a799dcdb2bd0ecb0273f2142efd, Thanks!